lib/tag/tag.js

```diff
@@ -28,7 +28,9 @@ function Tag(impl, conf, innerHTML) {
 
   // grab attributes
   each(root.attributes, function(el) {
-    attr[el.name] = el.value
+    var val = el.value
+    // remember attributes with expressions only
+    if (brackets(/\{.*\}/).test(val)) attr[el.name] = val
```

Comment:
`brackets(0)` defaults to `{`  `brackets(1)` to `}`

```javascript
var re = new RegExp(brackets(0)+'.*'+braclets(1))
if(re.test(...
```

Also the `.*` seems like it may not support multiple expressions?